clang/include/clang/Sema/Sema.h

```diff
@@ -11802,9 +11817,10 @@ class Sema final : public SemaBase {
                                      bool PartialOrdering,
                                      bool *StrictPackMatch);
 
+  SmallString<128> toTerseString(const NamedDecl &D) const;
```

Comment:
would love a comment that tells what 'terse string' means here.  'terse' is actually a pretty loaded word in C++, so explainations need to be pretty sizable here.

Reply:
Given this is used 2 times, i wonder if it's actually useful. especially in Sema.
If we really think we need thar, I'd rather have it be a new overload of Decl::print, or `Decl::printTerse` or something like that

Reply:
Added comment.

The helper is still useful as it returns a string we can pass directly to diagnostic engine, versus a print overload which should operate on streams, as otherwise it would be a bit user hostile for print to mean two different things.

I believe this helper will be temporary as I will remove its other user in a future patch.